db/models.py

```diff
@@ -50,3 +52,47 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
```

Comment:
Code Style: It's recommended to use `get_user_model()` instead of directly referring to the `User` model. This is considered a best practice in Django because it allows for flexibility if the user model is swapped out in the future.